server/ansible/roles/pbench-server-firewall/tasks/main.yml

```diff
@@ -3,6 +3,7 @@
   firewalld:
     service: "{{ item }}"
     permanent: yes
+    immediate: yes
     state: enabled
   with_items:
     - http
```

Comment:
Just as a note (doesn't need to be worked out now), but we're going to need to punch a hole for the API port too, currently 8001 by default -- but we should create a fact to pull this from the target config file.

I also wonder whether it'd be better to set up a proxy configuration that will let us split the /api URI path from other accesses and drive everything off http/https without using a separate port, which would make more sense.

(Further note, as we move forward to user authentication, we'll probably want to drop http entirely.)